packages/components/src/Components/FormElements/FormElements.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { storiesOf } from '@storybook/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { boolean, object, text } from '@storybook/addon-knobs/react';
+
+import Label from './Label';
+
+storiesOf('FormElements/Label', module).addWithJSX('Default', () => (
+  <Label
+    fieldName="ControlOne"
+    labelFor="asdf"
```

Comment:
I think you changed the prop at some point

```suggestion
    htmlFor="asdf"
```